exporter/sentryexporter/sentry_exporter_test.go

```diff
@@ -17,14 +17,13 @@ package sentryexporter
 import (
 	"context"
 	"encoding/hex"
-	"testing"
-
 	"github.com/getsentry/sentry-go"
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
 	"github.com/stretchr/testify/assert"
 	"go.opentelemetry.io/collector/consumer/pdata"
 	"go.opentelemetry.io/collector/translator/conventions"
+	"testing"
```

Comment:
Please undo the changes to imports here. Standard library imports are conventionally grouped together as the first group in an `import (...)` statement.

Reply:
Okay